Instagram/Models/Post.m

```diff
@@ -0,0 +1,55 @@
+//
+//  Post.m
+//  Instagram
+//
+//  Created by Sarah Wang on 7/7/21.
+//
+
+
+#import "Post.h"
+
+@implementation Post
+
+@dynamic postID;
+@dynamic userID;
+@dynamic author;
+@dynamic caption;
+@dynamic image;
+@dynamic likeCount;
+@dynamic commentCount;
+
+
++ (nonnull NSString *)parseClassName{
+    return @"Post";
+}
+
++ (void) postUserImage: ( UIImage * _Nullable )image withCaption: ( NSString * _Nullable )caption withCompletion: (PFBooleanResultBlock  _Nullable)completion {
+    
+    Post *newPost = [Post new];
+    newPost.image = [self getPFFileFromImage:image];
+    newPost.author = [PFUser currentUser];
+    newPost.caption = caption;
+    newPost.likeCount = @(0);
+    newPost.commentCount = @(0);
+    
+    [newPost saveInBackgroundWithBlock: completion];
+}
+
++ (PFFileObject *)getPFFileFromImage: (UIImage * _Nullable)image {
```

Comment:
nit: inconsistent spacing. Also for the name. I wouldn't use `get` it's a very "Java" style of writing code. It's generally assumed that it's a get from the return type and the name. ex: `pfFileFromImage` I would assume this returns a file and takes an image.